redhat-developer/developers.redhat.com
# [RHDX-86] Simplify eloqua assembly options

### JIRA Issue Link
[RHDX-86](https://projects.engineering.redhat.com/browse/RHDX-86)

The purpose of this PR is to remove an unused assembly type,  rename 'Sign up form' to 'Eloqua form', and verify the Eloqua Form can be used in fields where it is expected to be available. 

### Verification Process

- Verify the assembly type Eloqua Form is available on the content types Assembly Page, Article, Product, and Topic Page. There are 3 fields allowed on the Product node:  field_downloads_page_content, field_getting_started_content, field_content
- Verify the visual style 2 column works on the assembly type. (Use the Form field to reference an existing assembly named Oracle OpenJDK form)
- Verify the assembly type Eloqua Form Embed is not available to be used in the fields on the node types mentioned above.

Notes:
- There was an @todo in the rhdp2.theme file to move logic to the rhd_eloqua_form module, so I did that.
